content/docs/get-started/policy-as-code/authoring-a-policy-pack.md

```diff
@@ -0,0 +1,133 @@
+
+    ```typescript
+    // Create a new Policy Pack.
+    new PolicyPack("policy-pack-typescript", {
+        // Specify the Policies in the Policy Pack.
+        policies: [{
+            // The name for the Policy must be unique within the Pack.
+            name: "s3-no-public-read",
+
+            // The description should document what the Policy does and why it exists.
+            description: "Prohibits setting the publicRead or publicReadWrite   permission on AWS S3 buckets.",
+
+            // The enforcement level can either be "advisory" or "mandatory". An    "advisory" enforcement level
+            // simply prints a warning for users, while a "mandatory" policy will block     an update from proceeding.
+            enforcementLevel: "mandatory",
+
+            // One or more rules can be specified as part of a Policy.
+            rules: [
+                // The typedRule function allows you to filter resources. In this case,     the rule only
+                // applies to S3 buckets and asserts that the acl is not "public-read"  nor "public-read-write".
+                // If the assertion fails, the custom assertion message will be     displayed to users.
+                typedRule(aws.s3.Bucket.isInstance, it => assert.ok(it.acl !==  "public-read"
+                    && it.acl !== "public-read-write",
+                    "You cannot set public-read or public-read-write on an S3 bucket. "     +
+                    "Read more about ACLs here: https://docs.aws.amazon.com/AmazonS3/   latest/dev/acl-overview.html")),
+            ],
+        }],
+    });
+    ```
+
+    You can find more example Policy Packs in the [examples repo](https://github.com/pulumi/examples/tree/master/policy-packs). Best practices for writing a Policy Pack can be found [here](../best-practices).
+
+## Testing the Policy Pack Locally
+
+Policy Packs can be tested on a user’s local workstation to facilitate rapid development and testing of policies. This removes the step of publishing and applying policy packs to the Pulumi Console and lets developers reference a policy pack on their local workstation.
+
+1. Run `npm install` in the Policy Pack directory.
+
+1. Use the `--policy-pack` flag by going to a directory that contains a Pulumi stack. This flag allows for quicker feedback loops when developing a Policy Pack.
+
+    If you don’t have a stack readily available, you can create a new stack for testing by running `pulumi new aws-typescript` in an empty directory. This AWS example will create an S3 bucket, which is perfect for testing our Policy.
+
+    In the stack directory run:
+
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi preview --policy-pack    <path-to-new-policy-pack-directory>
+    ```
+
+    If the stack is in compliance, we expect the output to simply tell us which     Policy Packs were run.
+
+    ```sh
```

Comment:
Hugo has a bug with highlighting code fences in lists, which causes this particular block to render incorrectly:

<img width="763" alt="Screen Shot 2019-11-05 at 11 06 06 AM" src="https://user-images.githubusercontent.com/710598/68237647-5fefdb00-ffbc-11e9-9dad-8ed77bf2703d.png">

I generally prefer we keep using the "three backtick` code fences where we can, but I have found a workaround for the cases that don't render correctly.

Using this:

```
    {{< highlight sh >}}
$ PULUMI_DEBUG_COMMANDS=true pulumi preview --policy-pack ~/policy-pack-typescript
Previewing update (dev):

        Type                 Name          Plan
    +   pulumi:pulumi:Stack  test-dev  	create
    +   └─ aws:s3:Bucket     my-bucket     create

Resources:
    + 2 to create

Permalink:
...
{{< /highlight >}}
```

Note:
- The line containing `{{< highlight sh >}}` is indented, which keeps the whole block indented.
- The rest of the lines aren't indented (but they will be indented when rendered).

Will render it correctly as:

<img width="753" alt="Screen Shot 2019-11-05 at 11 08 09 AM" src="https://user-images.githubusercontent.com/710598/68237738-93326a00-ffbc-11e9-8e15-70ddafafd7b1.png">